string label
float64 timestamp
float64[] quaternions
float64[] accelerations
float64[] angular_velocities